src/bo4e/enum/ausschreibungsstatus.py

```diff
@@ -0,0 +1,13 @@
+# pylint: disable=missing-module-docstring
+from bo4e.enum.strenum import StrEnum
+
+
+class Ausschreibungsstatus(StrEnum):
+    """
+    Bezeichnungen für die Ausschreibungsphasen
+    """
+
+    PHASE1 = "PHASE1"  #: Phase1: Teilnahmewettbewerb
+    PHASE2 = "PHASE2"  #: Phase2: Angebotsphase
+    PHASE3 = "PHASE3"  #: Phase3: Verhandlungsphase
+    PHASE4 = "PHASE4"  #: Phase4: Zuschlagserteilung
```

Comment:
wow, what a great and meaningful enum ;)

Reply:
ja, da hat wohl nie jemand eine echte review gemacht ;)

Reply:
#224